src/pages/upgrades/get-involved/index.tsx

```diff
@@ -541,9 +541,7 @@ export const query = graphql`
     locales: allLocale(
       filter: {
         language: { in: $languagesToFetch }
-        ns: {
-          in: ["src-pages-upgrades-get-involved-index", "components", "common"]
-        }
+        ns: { in: ["page-upgrades-get-involved", "common"] }
```

Comment:
![Screen Shot 2023-02-15 at 11 02 11 PM](https://user-images.githubusercontent.com/15589226/219281599-894cf76e-9ada-4abd-8a59-2d16661af130.png)

This missing translation string is in `page-upgrades-index` which isn't imported here. Looks like it's only used in this file as well, so we could consider just moving this string. This string probably isn't needed anyways, Lodestar is now in production. 

I think 2 best options are:

1 - Move this `consensus-client-under-review` into `page-upgrades-get-involved`
2 - Make a PR to dev, which can be merged back into here, which removes this string and its use. Created an issue for this: https://github.com/ethereum/ethereum-org-website/issues/9498

Reply:
Good catch. Thanks for creating that issue. Probably, it will be easier to implement that PR and clean this after merging dev (option 2).

Once we are done with the review here, I'll update this branch with dev.

Reply:
https://github.com/ethereum/ethereum-org-website/pull/9537

Heres a PR for this